openmls/src/group/mls_group/staged_commit.rs

```diff
@@ -121,16 +121,15 @@ impl MlsGroup {
         }
 
         // Determine if Commit is own Commit
-        let sender_index = sender.to_leaf_index();
-        let is_own_commit = sender_index == provisional_tree.own_node_index();
+        let sender = mls_plaintext.sender_index();
```

Comment:
Comment for the lines above: If we were removed from the group, we should also return a diff that can be applied to the group in order to reflect the new membership state.

Reply:
Ah, right. That's not what's happening right now, though, is it? So it should be a FIXME, ideally with an issue number? Also, there is a trade-off here, as this would require us to allow a valid TreeSync state, where the `own_leaf_index` potentially points somewhere outside of the tree.

Reply:
I filed #616 for that.